package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "@standardnotes/snjs",
-  "version": "2.0.31",
+  "version": "2.0.32",
   "engines": {
-    "node": ">=14.0.0 <15.0.0"
+    "node": ">=14.0.0 <16.0.0"
```

Comment:
this still supports 14 as the minimum - should 15 be the minimum as the PR title says?

Reply:
True, I changed the title.